backend/FwLite/FwLiteProjectSync/CrdtFwdataProjectSyncService.cs

```diff
@@ -49,11 +49,15 @@ private async Task<SyncResult> Sync(IMiniLcmApi crdtApi, IMiniLcmApi fwdataApi,
 
         //todo sync complex form types, parts of speech, semantic domains, writing systems
 
+        var currentFwDataPartsOfSpeech = await fwdataApi.GetPartsOfSpeech().ToArrayAsync();
+        var crdtChanges = await PartOfSpeechSync(currentFwDataPartsOfSpeech, projectSnapshot.PartsOfSpeech, crdtApi);
+        var fwdataChanges = await PartOfSpeechSync(await crdtApi.GetPartsOfSpeech().ToArrayAsync(), currentFwDataPartsOfSpeech, fwdataApi);
+
         var currentFwDataEntries = await fwdataApi.GetEntries().ToArrayAsync();
-        var crdtChanges = await EntrySync(currentFwDataEntries, projectSnapshot.Entries, crdtApi);
+        crdtChanges += await EntrySync(currentFwDataEntries, projectSnapshot.Entries, crdtApi);
         LogDryRun(crdtApi, "crdt");
 
-        var fwdataChanges = await EntrySync(await crdtApi.GetEntries().ToArrayAsync(), currentFwDataEntries, fwdataApi);
+        fwdataChanges += await EntrySync(await crdtApi.GetEntries().ToArrayAsync(), currentFwDataEntries, fwdataApi);
         LogDryRun(fwdataApi, "fwdata");
```

Comment:
Currently crdtChanges and fwdataChanges are simple integers; perhaps they should become more complex types, so that a sync that changes 1 part of speech and 1 entry won't report 2 changes? (Because right now, FwHeadless is logging "2 entries changed" but will eventually need to log "1 part of speech changed and 1 entry changed".

Reply:
I'm fine with this as is, if they want to see what changed they can look in the history/activity log from FW Lite to see all the details of what happened.

Reply:
Okay, won't change anything here.